salt/cilium/init.sls

```diff
@@ -0,0 +1,16 @@
+include:
+  - ca-cert
+  - cert
+  - crypto
+
+{% set plugin = salt['pillar.get']('cni:plugin', 'cilium').lower() %}
+{% if plugin == "cilium" %}
+
+{% from '_macros/certs.jinja' import certs with context %}
+{{ certs("cilium",
+         pillar['ssl']['cilium_crt'],
+         pillar['ssl']['cilium_key'],
+         cn = grains['nodename'],
+         o = 'system:nodes') }}
+
+{% endif %}
```

Comment:
As a minor comment, I would move this folder from the top level to `cni/cilium` (and use `cni/cilium` instead of just `cilium` in `top.sls`).